client/scss/components/_promo.scss

```diff
@@ -280,7 +266,7 @@
     }
   }
 
-  @include container-query(('l': ('8', '12'))) {
+  @include container-query(('l': ('8', '12'), 'xl': ('8', '12'))) {
```

Comment:
This seems to be a common problem with the `between` concept on the media queries, similar to what we're fixing in this branch.

Should it not just be that the larger the breakpoint just overrides and you fallback to the one below, follow general convention of media queries and CSS?